test/constraints/constraint_connection.jl

```diff
@@ -820,5 +812,5 @@
             @test _is_constraint_equal(observed_con, expected_con)     
             return       
         end
-    end
+    end    
 end
```

Comment:
Thanks for fixing the tests. Would you consider writing tests for the new constraints? Maybe after we merge?

Reply:
Of course, that is the plan... I'm working on them